runtime-modules/content/src/lib.rs

```diff
@@ -4907,7 +4907,7 @@ decl_event!(
 
         // Rewards
         ChannelPayoutsUpdated(UpdateChannelPayoutsParameters, Option<DataObjectId>),
-        ChannelRewardUpdated(Balance, ChannelId),
+        ChannelRewardUpdated(Balance, Balance, ChannelId),
```

Comment:
Can we guarantee that the query node will not have processed a `ChannelRewardUpdated` event before ephesus release? Otherwise it is not safe to change the type this way. It would be safer to add the cashout_amount at the end of the tuple.

Reply:
Answering myself, reason I ask because the extrinsic `claim_channel_reward` is not filtered, but of course `set_channel_paused_features_as_moderator` is filtered, so it is not possible to "unpause" `CreatorCashout` feature prior to ephesus.